src/themes/themes.stories.mdx

```diff
@@ -15,7 +15,11 @@ While selector depth is best kept to a minimum, it's permissable to style object
 `t-dark` is intended primarily for page headers.
 
 <Preview>
-  <Story name="Dark" height="200px" parameters={{ themes: { disable: true } }}>
+  <Story
+    name="Dark"
+    height="200px"
+    parameters={{ themes: { disable: true }, paddings: { disabled: true } }}
```

Comment:
I'm probably missing it, but can't tell what the `paddings` property is changing here. Also, FYI (nit) see spelling `permissable` -> `permissible` in this file.

Reply:
It is disabling the paddings add-on for this story because it doesn't work with the theme preview.

I'll fix the typo 👍

Reply:
Just to clarify, I found this setting made a visible difference when I toggled it in the demo that's imported (`fill-canvas`), but couldn't see what difference it was making here, when I toggled it. I may have been missing it.